examples/getting-started.md

```diff
@@ -115,61 +116,104 @@ This section verifies that the network policies are correctly isolating applicat
+  kubectl exec -it $HELLOWORLD_POD_HS1 -n hs1 --kubeconfig=provider.conf -- apt update
+  kubectl exec -it $HELLOWORLD_POD_HS1 -n hs1 --kubeconfig=provider.conf -- apt install curl -y
+  
   # Test connectivity from hs1 to hs2 using the IP
-  kubectl exec -it $HELLOWORLD_POD_HS1 -n hs1 --kubeconfig=provider.conf -- curl $HS2_POD_IP
+  kubectl exec -it $HELLOWORLD_POD_HS1 -n hs1 --kubeconfig=provider.conf -- curl $HS2_POD_IP:5000
   ```
 
   The connection should be denied.
 
 - **Ping/HTTP Test from `hs2` to `hs1`:**
 
-  ```bash
+  ```sh
   # Get the Pod name for hs2
   HELLOWORLD_POD_HS2=$(kubectl get pods -n hs2 --kubeconfig=provider.conf -o jsonpath='{.items[0].metadata.name}')
-
+  
   # Get the Pod IP for hs1
   HS1_POD_IP=$(kubectl get pods -n hs1 --kubeconfig=provider.conf -o jsonpath='{.items[0].status.podIP}')
-
+  
+  # Update and install curl on hs2 pod
+  kubectl exec -it $HELLOWORLD_POD_HS2 -n hs2 --kubeconfig=provider.conf -- apt update
+  kubectl exec -it $HELLOWORLD_POD_HS2 -n hs2 --kubeconfig=provider.conf -- apt install curl -y
+  
   # Test connectivity from hs2 to hs1 using the IP
-  kubectl exec -it $HELLOWORLD_POD_HS2 -n hs2 --kubeconfig=provider.conf -- curl $HS1_POD_IP
+  kubectl exec -it $HELLOWORLD_POD_HS2 -n hs2 --kubeconfig=provider.conf -- curl $HS1_POD_IP:5000
   ```
 
   The connection should be denied.
 
-## Clean Up
+### Allowing Cross Namespace Traffic
+
+In some scenarios, you might want to enable controlled communication between instances running in different namespaces. KubePlus provides a custom kubectl plugin for this purpose. To allow bi-directional traffic between the two HelloWorldService instances (deployed in namespaces `hs1` and `hs2`), run:
+
+```sh
+kubectl allow network traffic hs1 hs2 -k provider.conf
+```
+
+```sh 
+# Test connectivity from hs1 to hs2 using the IP
+kubectl exec -it $HELLOWORLD_POD_HS1 -n hs1 --kubeconfig=provider.conf -- curl $HS2_POD_IP:5000
+
+# Test connectivity from hs2 to hs1 using the IP 
+kubectl exec -it $HELLOWORLD_POD_HS2 -n hs2 --kubeconfig=provider.conf -- curl $HS1_POD_IP:5000
+```
+
+The connection should be allowed
+
 
+To deny the traffic between namespace 
 
-```bash
-$ kubectl delete -f hs1-no-replicas.yaml --kubeconfig=provider.conf
-$ kubectl delete -f hs2-no-replicas.yaml --kubeconfig=provider.conf
-$ kubectl delete -f hello-world-service-composition.yaml --kubeconfig=provider.conf
+```sh
+kubectl deny network traffic hs1 hs2 -k provider.conf
 ```
 
+```sh 
+# Test connectivity from hs1 to hs2 using the IP
+kubectl exec -it $HELLOWORLD_POD_HS1 -n hs1 --kubeconfig=provider.conf -- curl $HS2_POD_IP:5000
+
+# Test connectivity from hs2 to hs1 using the IP 
+kubectl exec -it $HELLOWORLD_POD_HS2 -n hs2 --kubeconfig=provider.conf -- curl $HS1_POD_IP:5000
+```
+
+You should see that each policy’s ingress section now includes a rule that uses a namespaceSelector matching the other namespace (using the label `kubernetes.io/metadata.name`).
```

Comment:
We should move this comment up, before the statement "The connection should be allowed".

Also, can you include kubectl command to show the network policy object. Add it after the above comment.